packages/eui/.storybook/addons/code-snippet/decorators/render_jsx.tsx

```diff
@@ -272,11 +272,28 @@ export const renderJsx = (
 
     // ensure tokens are output properly by removing added variable markers
     if (string.indexOf('{{') > -1) {
-      const regex = new RegExp(/'{{|}}'/g);
+      const regex = new RegExp(/'{{|"{{|}}'|}}"/g);
       const matches = string.match(regex);
       if (matches) {
         matches.forEach((match) => {
-          string = string.replace(match, match.replace(regex, ''));
+          string = string.replace(match, '');
+        });
+      }
+    }
+
+    // removed arg value overwrite markers
+    // example:
+    // in:  selected: "#{moment('Tue Mar 19 2024 18:54:51 GMT+0100')}"
+    // out: selected={moment('Tue Mar 19 2024 18:54:51 GMT+0100')}
+    if (string.indexOf('#{') > -1) {
```

Comment:
Do we want startsWith instead here maybe? Not sure if that needs to be that specific?

```suggestion
    if (string.startsWith('#{')) {
```

Reply:
Ahaha, yeah sure. I still need to get used to thinking of using `startsWith` instead 🫠

Reply:
Ah wait no, it won't work here though because it does not actually start with that string. We're doing the match on the entire component jsx string.

Reply:
Ahhh okay! No worries at all, thanks for checking!